goth/runner/cli/yagna_payment_cmd.py

```diff
@@ -110,15 +104,15 @@ def payment_init(
 
     def payment_status(
```

Comment:
Same as above, if we decide to use `PaymentConfig` in these calls, let's change this one as well for consistency.